src/pages/komodo-defi-framework/api/legacy/sell/index.mdx

```diff
@@ -14,20 +14,20 @@ The `sell` method issues a sell request and attempts to match an order from the
+| ----------------- | ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| base              | string                                | the name of the coin the user desires to sell                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| rel               | string                                | the name of the coin the user desires to receive                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| price             | numeric string or rational            | the amount in `rel` the user wants per one unit of the `base` coin                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
> user wants per one unit

Same idea, "willing to receive per one unit"

Reply:
updated to `the amount of `rel` coins the user wants to receive for each unit of the `base` coin spent.` (same as setprice)